### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by marttamayy**

### CODE REVIEW

File: Assignment2/marttamayy-Marta Mayoral.ttl [hunk 1]
```diff
@@ -0,0 +1,7 @@
+:Class01 vocab:includes :Sensor029;
+		 vocab:includes :Computer101.
+:Sensor029 vocab:hasMeasurement :Measurement8401.
+:Measurement8401 vocab:hasTemperature :29;
```
Comment (ocorcho) [inline Assignment2/marttamayy-Marta Mayoral.ttl:4]: Syntax error. Will not parse. Same in the following lines

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Review by marttamayy — COMMENTED** [review]
I checked the .ttl document and I made the requested changes.

File: Assignment2/marttamayy-Marta Mayoral.ttl [hunk 2]
```diff
@@ -0,0 +1,15 @@
+@base <http://example.org/class>.
+@prefix data: <http://example.org/class#>.
+@prefix xsd: <http://www.w3.org/2001/XMLSchema>.
+
+:Class01 data:includes :Sensor029;
+	 data:includes :Computer101.
+
+:Sensor029 data:hasMeasurement :Measurement8401.
+
+:Measurement8401 data:hasTemperature :"29"^^xsd:float;
```
Comment (ocorcho) [inline Assignment2/marttamayy-Marta Mayoral.ttl:10]: there are still syntax errors here. Please parse this in a validator first.

**Review by ocorcho — CHANGES_REQUESTED** [review]

**Review by marttamayy — COMMENTED** [review]
I'm sorry, i sent the wrong one, this one shouldn't have any syntax error.

**Pull request merged**